Desktop/components/JASP/Widgets/FileMenu/OSFLogin.qml

```diff
@@ -150,6 +149,22 @@ FocusScope
 			}
 		}
 
+		Text
+		{
+			text			: qsTr("<sup>*</sup>")
```

Comment:
I don't think a qsTr is necessary here. Translators will not know really what they have to do with this.

Reply:
Copy paste all the way!

But you are right, I'll change it.